From eb5881be78d435d381c72db53e3cde793eff57d6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lu=C3=ADs=20Pereira?= <luis.artur.pereira@gmail.com>
Date: Wed, 8 Apr 2015 05:04:39 -0700
Subject: [PATCH 9/9] XdgDesktopFile: Handles NotShowIn correctly

An typo (a missing !) lead to a bug. Testing is on it's way.
---
 xdgdesktopfile.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xdgdesktopfile.cpp b/xdgdesktopfile.cpp
index c46a4a7..281ca20 100644
--- a/xdgdesktopfile.cpp
+++ b/xdgdesktopfile.cpp
@@ -1162,7 +1162,7 @@ bool XdgDesktopFile::isSuitable(bool excludeHidden, const QString &environment)
     if (keyFound)
     {
         QStringList s = value(key).toString().toUpper().split(QLatin1Char(';'));
-        if (!s.contains(env))
+        if (s.contains(env))
             return false;
     }
 
-- 
1.9.0

